src/ucp/dt/dt.h

```diff
@@ -24,8 +25,14 @@ typedef struct ucp_dt_state {
     size_t                        offset;  /* Total offset in overall payload. */
     union {
         struct {
-            uct_mem_h             memh;
-        } contig;
+            struct {
```

Comment:
maybe it is worth to have a single struct and use just one element for contig case?

Reply:
there is logic to use same lane for am/get and using single element makes processing complicated